exercises/anagram/test/test_anagram.c

```diff
@@ -173,62 +168,47 @@ void test_case_insensitive_anagrams(void)
       "radishes"
    };
 
+   char outputs[][MAX_STR_LEN] = {
+      "Carthorse"
+   };
+
    struct Vector vin = {
       inputs,
       sizeof(inputs) / MAX_STR_LEN
    };
 
-   char outputs[][MAX_STR_LEN] = {
-      "Carthorse"
+   struct Vector expected = {
+      outputs,
+      sizeof(outputs) / MAX_STR_LEN
    };
-   int outputs_len = sizeof(outputs) / MAX_STR_LEN;
-   qsort(outputs, outputs_len, sizeof(char) * MAX_STR_LEN, stringcmp);
-
-   struct Vector vout = anagrams_for("Orchestra", vin);
-   qsort(vout.vec, vout.size, sizeof(char) * MAX_STR_LEN, stringcmp);
-
-   TEST_ASSERT_EQUAL(outputs_len, vout.size);
-   int x;
-   for (x = 0; x < vout.size; x++) {
-      TEST_ASSERT_EQUAL(strlen(outputs[x]), strlen(vout.vec[x]));
-      TEST_ASSERT_EQUAL_MEMORY(outputs[x], vout.vec[x], strlen(outputs[x]));
-   }
 
-   free(vout.vec);
+   check_anagrams("Orchestra", vin, expected);
 }
 
 void test_unicode_anagrams(void)
 {
-   int x;
    // These words don't make sense, they're just greek letters cobbled together.
    char inputs[][MAX_STR_LEN] = {
       "ΒΓΑ",
       "ΒΓΔ",
       "γβα"
    };
 
+   char outputs[][MAX_STR_LEN] = {
+      "ΒΓΑ"
+   };
+
    struct Vector vin = {
       inputs,
       sizeof(inputs) / MAX_STR_LEN
    };
 
-   char outputs[][MAX_STR_LEN] = {
-      "ΒΓΑ",
-      "γβα"
```

Comment:
One of the outputs on this test shouldn't have been here, so I removed it. It was never hit in automation because the test is 'extra credit' and commented out.